migration/migrator/migrations/master/202306205342134_add_course_and_semester_constraint.py

```diff
@@ -0,0 +1,7 @@
+def up(config, database):
+    sql = "ALTER TABLE courses ADD CONSTRAINT no_surrounding_spaces_course CHECK (course ~ '^[^ ].*[^ ]$'); ALTER TABLE terms ADD CONSTRAINT no_surrounding_spaces_semester CHECK (term_id ~ '^[^ ].*[^ ]$');"
+    database.execute(sql)
+
+def down(config, database):
+    sql = "ALTER TABLE courses DROP CONSTRAINT no_surrounding_spaces_course; ALTER TABLE terms DROP CONSTRAINT no_surrounding_spaces_semester;"
+    database.execute(sql)
```

Comment:
These lines are pretty long, so it would probably be a good idea to break them into multiple lines (although still as part of a single string).